app/src/main/java/com/amaze/filemanager/utils/OTGUtil.java

```diff
@@ -47,7 +47,9 @@
 import androidx.documentfile.provider.DocumentFile;
 
 /** Created by Vishal on 27-04-2017. */
-public class OTGUtil {
+public abstract class OTGUtil {
```

Comment:
Adding abstract makes no sense, it has no abstract methods.

Reply:
Was habit to prevent others from instantiating utility classes... anyway.

Reply:
A private constructor would be a solution I would approve.